cpu/stm32/dist/spi_divtable/spi_divtable.c

```diff
@@ -22,6 +22,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 
+#define ARRAY_SIZE(a) (sizeof((a)) / sizeof((a)[0]))
```

Comment:
Any reason for redefining this and not reusing the definition from `core/include/kernel_defines.h`?

Reply:
The tool is not built by the RIOT build system, but by its own Makefile. In fact, before this PR the build of this tool is broken.

Reply:
Of course, thanks for the clarification!